### ISSUE TITLE
Beats 7.3 decode_base64_field is not working

### ISSUE THREAD

**Issue by guyromb** [elastic/beats#13143]
Seems like key is not readable when using decode_base64_field.

As described here: https://www.elastic.co/guide/en/beats/filebeat/master/decode-base64-field.html

I use the following processor as auto discovery hint:

```
processors:
- decode_base64_field:
    field:
      from: "a.b"
      to: "a.c"
``` 

**Result**: "failed to decode base64 fields in processor: could not fetch value for key: , Error: key not found"
**Expected**: fetching correct value using key field1, and output to field2.

**Comment by guyromb** [comment]
https://github.com/elastic/beats/pull/13144

**Comment by kvch** [comment]
This issue has been addressed in https://github.com/elastic/beats/pull/13092. But unfortunately, it did not made into 7.3. It's going to be released in 7.3.1.

**Comment by ilyaevseev** [comment]
The same problem with filebeat 7.8.0.

See also unanswered question for 7.6.1:
https://discuss.elastic.co/t/filebeat-base64-decode-key-not-found-7-6-1/223229
